source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1661,9 +1662,22 @@ unsigned SwiftExpressionParser::Parse(DiagnosticManager &diagnostic_manager,
         variable_map[name] = *var_info;
       }
 
+  llvm::SmallVector<std::pair<swift::FuncDecl *, std::vector<int>>, 4> FDAIs;
+  parsed_expr->code_manipulator->EnumerateDefaultValuedArguments(FDAIs);
+
   std::unique_ptr<swift::SILModule> sil_module(swift::performSILGeneration(
       parsed_expr->source_file, swift_ast_ctx->GetSILOptions()));
 
+  for (const auto &FDAI : FDAIs) {
+    swift::FuncDecl *FD = FDAI.first;
+    std::vector<int> DAI = FDAI.second;
+    for (int AI : DAI) {
+      auto DAG = swift::SILDeclRef::getDefaultArgGenerator(FD, AI);
+      swift::SILFunction *F = sil_module->lookUpFunction(DAG);
+      F->dump();
```

Comment:
I don't see how this has any effect (besides spewing garbage to stderr). What's actually going on here?

Reply:
I disagree with the spewing garbage bit at least - it was very useful to me when I was trying to determine if I was catching everything and the correct thing :-).  Yes, this is absolutely debugging cruft that is not meant to be merged.